vendor/sunnyvalley/+POST_INSTALL

```diff
@@ -0,0 +1,15 @@
+
+if [ ! -f /usr/local/sbin/opnsense-version ]; then
+	echo "OPNsense not installed, please have a proper OPNsense installation first"
+	exit 1
+fi
+
+ISLIBRESSL=$(/usr/local/sbin/opnsense-version | grep LibreSSL)
```

Comment:
opnsense-version -f

you should abort if it doesn't return the expected "OpenSSL" or "LibreSSL" and you can therefore remove the upper "not installed, etc"

Reply:
Got it.